ASP.NET WebApp (No Auth) for beta7

Feed used: https://www.myget.org/F/aspnetliterelease/api/v2 
 Samples are here: https://onedrive.live.com/redir?resid=43459A6389A804D9%21102054

With the new feed (http://myget.org/F/aspnetbeta7take2/api/v2) the restore works. However I am getting a compilation error on build:

	C:\DevProjects\VSCode\WebApplicationBasic\Startup.cs(43,27): DNXCore,Version=v5.0 error CS1061: 'ILoggerFactory' 
		does not contain a definition for 'AddDebug' and no extension method 'AddDebug' accepting a first argument of 
		type 'ILoggerFactory' could be found (are you missing a using directive or an assembly reference?)

Line 43 of Startup.cs.

	loggerFactory.AddDebug();

I think maybe the Web Basic template provided is missing a dependency in the project.json file.

	    "Microsoft.Framework.Logging.Debug" : "1.0.0-beta7"

--- a/templates/projects/webbasic/Startup.cs
+++ b/templates/projects/webbasic/Startup.cs
@@ -3,15 +3,11 @@ using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNet.Builder;
-using Microsoft.AspNet.Diagnostics;
 using Microsoft.AspNet.Hosting;
-using Microsoft.AspNet.Http;
-using Microsoft.AspNet.Routing;
+using Microsoft.Dnx.Runtime;
 using Microsoft.Framework.Configuration;
 using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Logging;
-using Microsoft.Framework.Logging.Console;
-using Microsoft.Framework.Runtime;
 
 namespace <%= namespace %>
 {
@@ -26,7 +22,7 @@ namespace <%= namespace %>
             Configuration = builder.Build();
         }
 
-        public IConfiguration Configuration { get; set; }
+        public IConfigurationRoot Configuration { get; set; }
 
         // This method gets called by the runtime.
         public void ConfigureServices(IServiceCollection services)
@@ -44,6 +40,7 @@ namespace <%= namespace %>
         {
             loggerFactory.MinimumLevel = LogLevel.Information;
             loggerFactory.AddConsole();
+            loggerFactory.AddDebug();
 
             // Configure the HTTP request pipeline.
 

--- a/templates/projects/webbasic/project.json
+++ b/templates/projects/webbasic/project.json
@@ -3,22 +3,22 @@
   "version": "1.0.0-*",
 
   "dependencies": {
-    "Microsoft.AspNet.Diagnostics": "1.0.0-beta6",
-    "Microsoft.AspNet.Mvc": "6.0.0-beta6",
-    "Microsoft.AspNet.Mvc.TagHelpers": "6.0.0-beta6",
-    "Microsoft.AspNet.Server.IIS": "1.0.0-beta6",
-    "Microsoft.AspNet.Server.Kestrel": "1.0.0-beta6",
-    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta6",
-    "Microsoft.AspNet.StaticFiles": "1.0.0-beta6",
-    "Microsoft.AspNet.Tooling.Razor": "1.0.0-beta6",
-    "Microsoft.Framework.Configuration.Json": "1.0.0-beta6",
-    "Microsoft.Framework.Logging": "1.0.0-beta6",
-    "Microsoft.Framework.Logging.Console": "1.0.0-beta6",
-    "Kestrel": "1.0.0-beta6"
+    "Microsoft.AspNet.Diagnostics": "1.0.0-beta7",
+    "Microsoft.AspNet.Mvc": "6.0.0-beta7",
+    "Microsoft.AspNet.Mvc.TagHelpers": "6.0.0-beta7",
+    "Microsoft.AspNet.Server.IIS": "1.0.0-beta7",
+    "Microsoft.AspNet.Server.Kestrel": "1.0.0-beta7",
+    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta7",
+    "Microsoft.AspNet.StaticFiles": "1.0.0-beta7",
+    "Microsoft.AspNet.Tooling.Razor": "1.0.0-beta7",
+    "Microsoft.Framework.Configuration.Json": "1.0.0-beta7",
+    "Microsoft.Framework.Logging": "1.0.0-beta7",
+    "Microsoft.Framework.Logging.Debug" : "1.0.0-beta7",
+    "Microsoft.Framework.Logging.Console": "1.0.0-beta7"
   },
 
   "commands": {
-    "kestrel": "Microsoft.AspNet.Hosting --server Kestrel --config hosting.ini",
+    "kestrel": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.Kestrel --config hosting.ini",
     "web": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.WebListener --config hosting.ini"
   },
 

